STACpopulator/extensions/datacube.py

```diff
@@ -248,3 +248,10 @@ def temporal_extent(self) -> MutableSequence[str]:
         start_datetime = cfmeta["time_coverage_start"]
         end_datetime = cfmeta["time_coverage_end"]
         return [start_datetime, end_datetime]
+
```

Comment:
Should this class be a subclass of `ExtensionHelper`?

Reply:
Yes, I think it would be good to have a clear hierarchy, but I didn't want to break too much stuff in one PR.